packages/bridge/contracts/l2/L2DaiGateway.sol

```diff
@@ -147,12 +126,12 @@ contract L2DaiGateway is L2CrossDomainEnabled {
     bytes memory data
   ) public view returns (bytes memory outboundCalldata) {
     outboundCalldata = abi.encodeWithSelector(
-      ITokenGateway.finalizeInboundTransfer.selector,
+      L1ITokenGateway.finalizeInboundTransfer.selector,
       token,
       from,
       to,
       amount,
-      abi.encode(exitNum, data)
+      abi.encode(0, data)
```

Comment:
I'd add a comment about this 0 being encoded, otherwise looks to be a random magic value

Reply:
You could also just remove this altogether, but it different data encodings here would make it more annoying for subgraphs trying to index multiple gateways

Reply:
We plan on updating the messaging between our gateways in the future, so I guess this will need to be sepcial cased regardless. So I'd say you could drop encoding this 0 here and simplify this logic further